plugins/languages/rustaceanvim.nix

```
@@ -278,5 +274,18 @@ in {
         };
       }
       // cfg.extraOptions;
-  };
+  in
+    mkIf cfg.enable
+    {
+      extraPlugins = [cfg.package];
+
+      extraPackages =
+        optional
+        (cfg.rustAnalyzerPackage != null)
+        cfg.rustAnalyzerPackage;
+
+      plugins.lsp.postConfig = ''
+        vim.g.rustaceanvim = ${helpers.toLuaObject globalOptions}
+      '';
```

```suggestion
      plugins.lsp.postConfig = let
        globalOptions = with cfg; {
          ...
        };
      in ''
        vim.g.rustaceanvim = ${helpers.toLuaObject globalOptions}
      '';
```

I would rather preserve the "scope locality".